src/integrations/__spec__/testAccess.ts

```diff
@@ -170,6 +170,28 @@ export default function testClass(integrationName, integration) {
         expect(typeof Madrone.lastAccessed(person.personInner)).toEqual('number');
         expect(typeof Madrone.lastAccessed(personInner)).toEqual('number');
       });
+
+      test('computed that depends on reactive array that starts out with zero length', async () => {
```

Comment:
Looks like the other tests in this file are specifically testing the `lastAccessed` method. I think this test would be a better candidate for `testClass.ts`

Reply:
thanks, it's moved!